package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prettier-plugin-organize-imports",
-  "version": "1.1.1",
+  "version": "1.2.1",
```

Comment:
Please undo this change, version will be bumped when I actually do a release (also it would be `1.2.0`).

Reply:
Done!